### ISSUE TITLE
Outdated `:-moz-focusring` makes form elements less accessible

### ISSUE THREAD

**Issue by LeoniePhiline** [sindresorhus/modern-normalize#55]
The package applies an outdated inaccessible focus ring to form elements in firefox:

```css
/**
Restore the focus styles unset by the previous rule.
*/

:-moz-focusring {
	outline: 1px dotted ButtonText;
}
```

This thin dotted gray line is inaccessible, outdated. Browsers have updated their default focus styles to be much more accessible, including Firefox.

Here an unstyled, focused textarea (without `modern-normalize`):

![image](https://user-images.githubusercontent.com/22329650/123082859-c9d22800-d41f-11eb-8018-5cdf0875f38e.png)

https://codepen.io/rickymetz/pen/ZrbXqJ

It would be fantastic if modern-normalize could undo the custom application of an inaccessible legacy Firefox focus style.

**Comment by sindresorhus** [comment]
Are you sure https://github.com/sindresorhus/modern-normalize/blob/1fc6b5a86676b7ac8abc62d04d6080f92debc70f/modern-normalize.css#L195-L202 can be safely removed without introducing any regressions?

**Comment by LeoniePhiline** [comment]
Hi, thanks for your response!

I believe this can be removed. 

If you use Firefox to view the two-buttons example at https://developer.mozilla.org/en-US/docs/Web/CSS/::-moz-focus-inner you will see that it does not work any more. 

The `::-moz-focus-inner` styles apparently have no influence on Firefox's "new style" focus ring any more, as far as I can see.

**Comment by LeoniePhiline** [comment]
Update:
The related tailwindcss bug https://github.com/tailwindlabs/tailwindcss/issues/4769 has been fixed 🎉

Findings which might be just as relevant here in `modern-normalize`: https://github.com/tailwindlabs/tailwindcss/discussions/4724

**Comment by LeoniePhiline** [comment]
This might be something for `modern-normalize` to add a workaround for: https://github.com/tailwindlabs/tailwindcss/issues/4769#issuecomment-867728491

**Comment by atjn** [comment]
> This might be something for `modern-normalize` to add a workaround for: [tailwindlabs/tailwindcss#4769 (comment)](https://github.com/tailwindlabs/tailwindcss/issues/4769#issuecomment-867728491)

I can no longer reproduce that issue, I belive it is fixed now.
